### DISCUSSION TITLE
LXQT and WM

### DISCUSSION THREAD

**Discussion by CyberTatoka** [lxqt/lxqt#2589] [Q&A]
Hello, dear LXQT community! I want to ask you about WM of LXQT. What is optimal and standard WM for LXQT or LXQT is not related to a certain WM? Can I connect LXQT with Kwin, Sway or Hyprland?

**Comment by stefonarch** [comment]
https://github.com/lxqt/lxqt/wiki/ConfigWindowManagers shows an overview for X11.

For the wayland-session it will be the same - basically any wlroots-based compositor can be used and kwin_wayland, but you will have to set the compositor in a different section, the actual setting of the WM is for X11 only.  Preview:

![immagine](https://github.com/user-attachments/assets/055986e5-93df-458b-ba55-fdd752989b14)

**Reply by CyberTatoka** [reply]
Thank you, I have an one more question: Can any serious problems be with LXQt and Kwin for system and system's and personal important data? If so, what are they? And how much is it dangerous?

**Reply by stefonarch** [reply]
Until now no problems were reported.
